avm/res/kubernetes-configuration/flux-configuration/main.bicep

```diff
@@ -69,7 +69,7 @@ resource managedCluster 'Microsoft.ContainerService/managedClusters@2022-07-01'
   name: clusterName
 }
 
-resource fluxConfiguration 'Microsoft.KubernetesConfiguration/fluxConfigurations@2022-03-01' = {
```

Comment:
@JPEasier found out that with the new API version, the `kustomizations` become mandatory and that's why it is failing (even without an error message). 

Snippet:
```bicep
kustomizations: {
  unified: {
    dependsOn: []
    force: false
    path: './cluster-manifests'
    prune: false
    syncIntervalInSeconds: 300
    timeoutInSeconds: 300
  }
}
```